README.md

```diff
@@ -68,96 +52,121 @@ you may receive this error:
+like ``http://192.168.99.100:8079``
+- **Redirect uri**: Your **Url** value with "/complete/edxorg/" at the end
 
-Open an iPython shell and execute these commands:
+#### 5) Copy relevant values to use in the Micromasters .env file
 
-      edxapp@precise64:~/edx-platform$ python manage.py lms --settings=devstack shell
-      Python 2.7.10 (default, Jun 29 2015, 22:38:23)
-      Type "copyright", "credits" or "license" for more information.
+The Micromasters codebase contains a ``.env.sample`` file which will be used as
+a template to create your ``.env`` file. For Micromasters to work, it needs 3 values:
 
-      IPython 4.1.2 -- An enhanced Interactive Python.
-      ?         -> Introduction and overview of IPython's features.
-      %quickref -> Quick reference.
-      help      -> Python's own help system.
-      object?   -> Details about 'object', use 'object??' for extra details.
+- ``EDXORG_BASE_URL``
 
-      In [1]: from django.contrib.auth.models import User
+    This value _should_ be ``http://192.168.33.10:8000``. The Vagrant VM IP is hard-coded
+    in the Vagrantfile, and it's unlikely that edX will change that. The LMS server runs
+    on port ``8000`` by default.
+- ``EDXORG_CLIENT_ID`` and ``EDXORG_CLIENT_SECRET``
 
-      In [2]: User.objects.get(username='staff')
-      Out[2]: <User: staff>
+    These values can be found in Django admin OAuth client section discussed above.
+    **Client id:** and **Client secret:** values should be auto-generated for
+    your new client. Use those values for the corresponding ``EDXORG_``
+    variables in the ``.env`` file
 
-      In [3]: staff=_
+## Docker Container Configuration and Start-up
 
-      In [4]: staff.is_superuser=True
+First, create a ``.env`` file from the sample in the codebases:
 
-      In [5]: staff.save()
+    cp .env.sample .env
 
-      In [6]:
-      Do you really want to exit ([y]/n)? y
+Set the ``EDXORG_BASE_URL``, ``EDXORG_CLIENT_ID``, and ``EDXORG_CLIENT_SECRET``
+variables in the ``.env`` file appropriately.
 
-Start the LMS:
+For first-time container start-up, start it with a full build:
 
-    paver devstack lms
+    docker-compose up --build
 
-Navigate to ``localhost:8000/admin`` and login with the superuser account.
-Scroll down to the "Oauth2" section and open "Clients".
-Create a new client
-    name - your choice of name, mine is ``micromasters-local``.
-    Url - the URL of your local Micromasters instance, a URL on port ``8087``.
-    Redirect url: the same URL, but with ``/complete/edxorg/`` at the end. 
-    Client type: Chose "Confidential (Web applications)"
+In another terminal tab, navigate the the Micromasters directory
+and add a superuser in the now-running Docker container:
 
-Make a note of the Client id and the Client secret; you need them to
-configure Micromasters.  Don't forget to save your new client.
+    docker-compose run web python manage.py createsuperuser
```

Comment:
i just poked around to see where the docker container is being provisioned with python. looks like it's specified in ./apt.txt (`python3-dev`) and it's being installed via apt-get. i wonder if installing with apt-get fails to link `python` to the `python3` command, which homebrew apparently does for you. @blarghmatey any thoughts?